### DISCUSSION TITLE
Can I transfrom my Student Pack to another account

### DISCUSSION THREAD

**Discussion by wosk0x01** [community/community#39060] [GitHub Education] [answered]
For some reason, I need to change my account, but there seems to be no place to transfer StudPack from this account to another account. In my opinion, as long as we can verify the same school email address, we should be allowed to transfer the packs.

**Comment by hack1444** [comment]
YES ACCEPT İN

**Comment by wosk0x01** [comment, marked as answer]
according to the github support tickets, the answer is NO
kind of disappointing 😔
but I accept this result ::
